### PR TITLE
job: migrate ci-kubernetes-storage-scalability to community cluster

### PR DESCRIPTION
**Pull request by alessandrocapanna**
This PR move the ci-kubernetes-storage-scalability job to community cluster

@ameukam

Part of #31789

### CODE REVIEW

**Comment by alessandrocapanna** [comment]
@ameukam fixed as u said :D

File: config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml [hunk 1]
```diff
@@ -47,8 +48,13 @@ periodics:
...
       - --timeout=300m
-      - --use-logexporter
-      - --logexporter-gcs-path=gs://sig-scalability-logs/$(JOB_NAME)/$(BUILD_ID)
+      resources:
+        requests:
+          cpu: 2
+          memory: "2Gi"
+        limits:
+          cpu: 2
+          memory: "2Gi"
```
Comment (ameukam) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml:57]:
```suggestion
      resources:
        requests:
          cpu: 2
          memory: "4Gi"
        limits:
          cpu: 2
          memory: "4Gi"
```
Sorry I missed this. this a little lot of the test. We may to monitor the test post-migration.

Comment (alessandrocapanna) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml:57]: Yes maybe "2Gi" is already too little, I'm changing it now.

Comment (ameukam) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml]: Thanks!

**Comment by ameukam** [comment]
/lgtm
/approve

Please monitor the job for a few days: https://testgrid.k8s.io/sig-scalability-experiments#storage

**Pull request merged**